app/helpers/application_helper.rb

```diff
@@ -44,6 +44,17 @@ def auth_path(provider)
   def social_network_localize(provider)
     I18n.t("social_networks.#{provider}")
   end
+  
+  def flash_show_errors
+    tags = ""
+    if flash[:errors].present?
+      flash[:errors].each do |attr, val|
+        translate_path = "#{flash[:model_name_class]}.human_attribute_name(:#{attr})"
+        tags += content_tag :div, "Поле \"#{eval(translate_path)}\" #{val.to_s[2..-3]}" , class: "alert alert-dismissible alert-danger" 
```

Comment:
The use of `eval` is a serious security risk.<br>Line is too long. [136/80]<br>Space found before comma.<br>Trailing whitespace detected.